controls/radrichtextbox/document-elements/features-floatinguicontainer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: FloatingUIContainer
```

Comment:
Using API names for documentation headers and titles is not a good practice. We should use more descriptive names. In this case, I can see that the naming convention of the other articles was followed and also there is not a very good name for the article, but at least we can think of making it a bit more reading-friendly. For example: "Floating Container" or "Floating UI Container".